containers/docker/Dockerfile

```diff
@@ -1,101 +1,115 @@
-RUN curl https://raw.githubusercontent.com/kpeeples/kestrel-as-a-service/main/dockerhub/Rprofile.site -o ${HOME}/Rprofile.site
-RUN chown ${NB_UID}:${NB_GID} ${HOME}/Rprofile.site && mkdir -p /opt/conda/lib/R/etc && mv Rprofile.site /opt/conda/lib/R/etc/Rprofile.site
-#COPY --chown=${NB_UID}:${NB_GID} Rprofile.site /opt/conda/lib/R/etc/
-
-# Install kestrel-ANALYTICS
+RUN curl https://raw.githubusercontent.com/kpeeples/kestrel-as-a-service/main/dockerhub/Rprofile.site -o /home/jovyan/Rprofile.site
+RUN chown ${NB_UID}:${NB_GID} /home/jovyan/Rprofile.site && mkdir -p /opt/conda/lib/R/etc && mv Rprofile.site /opt/conda/lib/R/etc/Rprofile.site
+# from postbuild
+# https://github.com/opencybersecurityalliance/kestrel-huntbook/blob/main/.binder/postBuild
+################################
+# 3. Obtain Data -- This copies all data and notebooks required
+# Place data in /opt/kaas and then move to home for 
+# KaaS (Kubernetes/Jupyterhub/Containers)
+################################
 RUN git clone https://github.com/opencybersecurityalliance/data-bucket-kestrel.git /opt/kaas/data-bucket-kestrel
-
-RUN mkdir -p /opt/kaas/analytic-tutorials && \
-    mkdir -p /opt/kaas/huntbooks && \
-    mkdir -p /opt/kaas/tutorial && \
-    mkdir -p /opt/kaas/blackhat22
-
-# for generic huntbooks like BH22
-RUN cp -r /opt/kaas/data-bucket-kestrel/stix-bundles /opt/kaas/analytic-tutorials
-
-# for the tutorial
-RUN cp /opt/kaas/data-bucket-kestrel/stix-bundles/lab101.json /opt/kaas/analytic-tutorials
-
-RUN git clone https://github.com/opencybersecurityalliance/kestrel-analytics.git /opt/kaas/kestrel-analytics
-
-RUN cp /opt/kaas/data-bucket-kestrel/GeoLite2/GeoLite2-City.mmdb /opt/kaas/kestrel-analytics/analytics/piniponmap
-
-RUN rm -rf "/opt/kaas/data-bucket-kestrel"
-
-# install kestrel-HUNTBOOK
 RUN git clone https://github.com/opencybersecurityalliance/kestrel-huntbook.git /opt/kaas/kestrel-huntbook
-
-RUN mkdir -p /home/jovyan/.config/kestrel && \
-    mv /opt/kaas/kestrel-huntbook/config/stixshifter.yaml /home/jovyan/.config/kestrel/ && \
-    ln -s /home/jovyan/.config/kestrel/stixshifter.yaml /opt/kaas/kestrel-huntbook/huntbooks/stixshifter.yaml && \
-    ln -s /home/jovyan/.config/kestrel/stixshifter.yaml /opt/kaas/kestrel-huntbook/tutorial/stixshifter.yaml && \
-    ln -s /home/jovyan/.config/kestrel/stixshifter.yaml /opt/kaas/kestrel-huntbook/blackhat22/stixshifter.yaml
-
-RUN rmdir /opt/kaas/kestrel-huntbook/config
-
+RUN git clone https://github.com/opencybersecurityalliance/kestrel-analytics.git /opt/kaas/kestrel-analytics
+RUN ln -s /opt/kaas/data-bucket-kestrel /home/jovyan/data-bucket-kestrel 
+RUN ln -s /opt/kaas/kestrel-huntbook /home/jovyan/kestrel-huntbook 
+RUN ln -s /opt/kaas/kestrel-analytics /home/jovyan/kestrel-analytics 
+RUN cp /opt/kaas/data-bucket-kestrel/stix-bundles/lab101.json /tmp
+RUN cp /opt/kaas/data-bucket-kestrel/GeoLite2/GeoLite2-City.mmdb /opt/kaas/kestrel-analytics/GeoLite2-City.mmdb
```

Comment:
Oops, the destination dir should be `/opt/kaas/kestrel-analytics/analytics/piniponmap`.

Reply:
modified